messages.rb

```diff
@@ -64,6 +64,11 @@ def parse_rank(msg)
   rank ? rank.to_i : nil
 end
 
+def parse_bottom_rank(msg)
+  rank = msg[/bottom\s*([0-9][0-9]?)/i, 1]
+  rank ? 20-rank.to_i : nil
```

Comment:
Generally the style in inne is to leave spaces around binary operators like this, so `20 - rank.to_i`

Reply:
Fixed! I'll keep it in mind from now on.